plugins/GSdx/GS.cpp

```diff
@@ -476,7 +485,7 @@ EXPORT_C_(int) GSopen2(void** dsp, uint32 flags)
 		}
 
 #endif
-#if defined(__unix__)
+#if defined(__unix__) || defined(__APPLE__)
```

Comment:
Maybe setting Null renderer as default for macos would be better too.

Reply:
At the moment GSopen2 fails if it can't make a window (and the GL window considers itself failed if your GL implementation doesn't at least meet the requirements of the software renderer) so it doesn't really matter